ui/src/routes/organizations/$orgId/products/$productId/repositories/$repoId/-components/analyzer-fields.tsx

```diff
@@ -62,6 +92,114 @@ export const AnalyzerFields = ({
     control: form.control,
   });
 
+  const user = useUser();
+
+  // Only fetch infrastructure services the user has access to.
+  const infrastructureServices = useQueries({
```

Comment:
Note to myself: This basically does what was already proposed [here](https://github.com/eclipse-apoapsis/ort-server/pull/3829/commits/f4634cbc6182d18d2292507fa8c15dafa6285ab1#diff-736ea7708e9f894b850a802c39e16885d4b30ee9b0a4de0a4b6ab4982302afd9R91).